src/main/java/net/authorize/sample/TransactionReporting/GetAccountUpdaterJobDetails.java

```diff
@@ -0,0 +1,178 @@
+package net.authorize.sample.TransactionReporting;
+
+import java.util.ArrayList;
+import net.authorize.Environment;
+import net.authorize.api.contract.v1.ANetApiResponse;
+import net.authorize.api.contract.v1.AUJobTypeEnum;
+import net.authorize.api.contract.v1.AuDeleteType;
+import net.authorize.api.contract.v1.AuDetailsType;
+import net.authorize.api.contract.v1.AuUpdateType;
+import net.authorize.api.contract.v1.CreditCardMaskedType;
+import net.authorize.api.contract.v1.GetAUJobDetailsRequest;
+import net.authorize.api.contract.v1.GetAUJobDetailsResponse;
+import net.authorize.api.contract.v1.MerchantAuthenticationType;
+import net.authorize.api.contract.v1.MessageTypeEnum;
+import net.authorize.api.contract.v1.Paging;
+import net.authorize.api.controller.GetAUJobDetailsController;
+import net.authorize.api.controller.base.ApiOperationBase;
+
+public class GetAccountUpdaterJobDetails {
+
+	public static ANetApiResponse run(String apiLoginId, String transactionKey)
+			throws IllegalArgumentException, IllegalAccessException, NoSuchFieldException, SecurityException {
```

Comment:
Why are so many exceptions being expected from this method? There should be no throws in this method call.

Reply:
Changes have been done as per expectation.